.github/workflows/update_goldens.yaml

```
@@ -41,6 +40,6 @@ jobs:
       - name: 🚀 Commit updated golden files
         uses: stefanzweifel/git-auto-commit-action@v6.0.1
         with:
-          branch: ${{ inputs.branch }}
+          branch: ${{ inputs.branch || github.ref_name }}
           file_pattern: "packages/world_flags/test/goldens/*"
-          commit_message: "chore(test): update golden test files for `world_flags` package"
+          commit_message: "chore(test): update golden test files for `world_flags` package"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)